nx.json

```diff
@@ -11,6 +11,10 @@
     "build": {
       "dependsOn": ["^build"]
     },
+    "preview": {
+      "//": "Serves the app artifact created in the build target",
+      "dependsOn": ["build"]
+    },
```

Comment:
e.g. `nx run adknowledgeportal:preview` (runs `vite preview`, which serves the `build` folder) depends on `nx run adknowledgeportal:build`.